### DISCUSSION TITLE
client_secrets.json file not found Error at startup

### DISCUSSION THREAD

**Discussion by wuser200** [ThioJoe/YT-Spammer-Purge#440] [Ideas]
I had a json file named client_secrets.json and made a Google Cloud Platform Project and it's showing me this:

                  ----- [!] Error: client_secrets.json file not found -----
         ----- Did you create a Google Cloud Platform Project to access the API? -----
         > For instructions on how to get an API key, visit: www.TJoe.io/api-setup

         > (Non-shortened Link: https://github.com/ThioJoe/YT-Spammer-Purge/wiki/Instructions:-Obtaining-an-API-Key)

         Press Enter to Exit...

Please tell me how to fix this

**Comment by Kurdakov** [comment]
try to move to another user owned directory, probably some errors are hidden, but one error was just due to permission problems, moving application to another user owned directory might solve this

**Reply by KendallDoesCoding** [reply]
If this doesn't work, please create a issue thread @wuser200

**Reply by wuser200** [reply]
I don't know how to do that

**Comment by ThioJoe** [comment]
Moving discussion to here: https://github.com/ThioJoe/YT-Spammer-Purge/issues/481
